dix/req.h

```diff
@@ -253,6 +253,8 @@ struct m0_dix_req {
 	 * starting key in DIX_NEXT request.
 	 */
 	uint32_t                     *dr_recs_nr;
+	/** Minimum successful CAS operations. */
```

Comment:
Could you please use the more descriptive comment like
```
/**
  * Minimum number of successful CAS operations to treat
  * parent DIX operation as successful.
  * /
```
Current comment understand only those who are discussing that feature.

Reply:
Changed